server/models/display-error.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   'invalid-location': {
     errorCode: 'invalid-location',
+    view: 'find-location',
```

Comment:
It seems this model of display errors needs a view setting.  we set the yar variable on location error for displayError and locationError.  However I think we only need one of these settings, they don't seem to function separate to each other

Reply:
In fact removing the yar locationError cookie altogether as it was superfluous